src/pykeen/models/unimodal/pair_re.py

```diff
@@ -35,34 +38,56 @@ class PairRE(ERModel):
     def __init__(
         self,
         embedding_dim: int = 200,
-        p: int = 2,
-        power_norm: bool = True,
-        entity_initializer: Hint[Initializer] = xavier_normal_,
-        relation_initializer: Hint[Initializer] = xavier_normal_,
+        p: int = 1,
+        power_norm: bool = False,
+        entity_initializer: Hint[Initializer] = uniform_,
+        entity_initializer_kwargs: Optional[Mapping[str, Any]] = None,
+        entity_normalizer: Hint[Normalizer] = functional.normalize,
+        entity_normalizer_kwargs: Optional[Mapping[str, Any]] = None,
+        relation_initializer: Hint[Initializer] = uniform_,
+        relation_initializer_kwargs: Optional[Mapping[str, Any]] = None,
         **kwargs,
     ) -> None:
         r"""Initialize PairRE via the :class:`pykeen.nn.modules.PairREInteraction` interaction.
 
         :param embedding_dim: The entity embedding dimension $d$. Defaults to 200. Is usually $d \in [50, 300]$.
         :param p: The $l_p$ norm. Defaults to 2.
```

Comment:
fix the docs to show defaults to 1

Reply:
I removed it, since the default is already part of the method signature.
d6b4ba4